scripts/env_matrix.yml

```diff
@@ -4,6 +4,7 @@ CONDA_PY:
   - "3.5"
 CONDA_BOOST: "1.60"
 CONDA_PERL: "5.22.0"
+iCONDA_R: "3.3.0"
```

Comment:
No we don't. We are blocking on https://github.com/conda/conda-recipes/issues/683
It will be released this week hopefully.